apps/fabric-website/src/pages/Styles/TypographyPage/docs/web/TypographyCustomization.md

```diff
@@ -0,0 +1,14 @@
+You can add some of your own customizations to the default theme.
+For example, you can utilize `defaultFontStyle` to override the default font style and
+`fonts` to override the default medium font size.
```

Comment:
This wording probably shouldn't be specific to medium (though using just medium in the example should be fine), and you might note that you can change the weight (and possibly other properties?) as well.